test/_init_tests.js

```diff
@@ -188,6 +188,11 @@ afterEach(() => {
     sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
     throw new Error('You forgot to restore your sandbox!');
   }
+  if (global.sandbox) {
```

Comment:
Basically, some tests (they are fixed in this PR) would do things like `sandbox = sinon.sandbox.create()` but would not declare `let sandbox`, thus setting it on `global`.